hack/collector-default

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+HOST_PATH=$1
+BUNDLE_DIR=$2
+
+cd ${BUNDLE_DIR}
+
+###############################################################################
+# host information
+###############################################################################
+mkdir -p hostinfos
+cd hostinfos
+
+echo "hostname : " `cat ${HOST_PATH}/etc/hostname` > hostinfo
+echo "system_info : " `uname -a` >> hostinfo
+
+# Boot Configuration
+cat ${HOST_PATH}/boot/config-$(uname -r) > boot_config
+
+echo "HOST_PATH: " ${HOST_PATH} > tmp
+
+###############################################################################
+# collect logs
+###############################################################################
+cd ${BUNDLE_DIR}
+mkdir -p logs
+cd logs
+
+dmesg -HTx &> dmesg.log
+
+# syslog
+tail -c 10m ${HOST_PATH}/var/log/syslog > syslog.log
+
+# kernel log
+chroot ${HOST_PATH} /usr/bin/journalctl -k > kernel.log
+
+# kubelet log
+# On Linux nodes that use systemd, the kubelet and container runtime write to journald by default. 
+# ref. https://kubernetes.io/docs/concepts/cluster-administration/logging/#log-location-node
+chroot ${HOST_PATH} /usr/bin/journalctl -r -u kubelet > kubelet.log
+# RKE2: Logs for the kubelet are save in rke2 agent log folder
+RKE2_KUBELET_LOG_PATH=${HOST_PATH}/var/lib/rancher/rke2/agent/logs/kubelet.log
+if [ -f ${RKE2_KUBELET_LOG_PATH} ]; then
+    cp ${RKE2_KUBELET_LOG_PATH} .
+fi
```

Comment:
For non-rke2 env, this is invalid.

Suggest using the pattern for making the flow clear.
```
func collect_sysinfo()
{
    uname ...
}

func collect_dmesg()
{
  ...
}

func collect_kubelet()
{
  Iterate default k8s kubelet and do the best to collect the log 
}

...

collect_sysinfo
colect_dmesg
collect_kublete
...

```

Reply:
I have tried to separate into functions according to different purposes. Thank you.